wiki/Help_Centre/Restricted_Mode/en.md

```diff
@@ -0,0 +1,60 @@
+# Account Restrictions
```

Comment:
Change the folder name to `Account_Restrictions`.

Reply:
this still hasn't been done